web/package.json

```diff
@@ -7,7 +7,7 @@
     "wasm": "wasm-pack build --dev --target web ../osm2streets-js",
     "wasm-release": "wasm-pack build --release --target web ../osm2streets-js",
     "dev": "vite",
-    "build": "vite build",
+    "build": "npm run wasm-release && vite build",
```

Comment:
I just pushed a commit to this branch with a change to GH config due to this, FYI